src/templates/next/components/complex/Hero/Hero.tsx

```diff
@@ -86,7 +86,7 @@ const HeroSplit = ({
   return (
     <>
       <section
-        className={`hidden md:flex h-[32rem] ${
+        className={`hidden md:flex h-auto sm:h-[22.5rem] md:h-[31.25rem] ${
```

Comment:
issue: I think the rest needs to get a min-height as well?

Reply:
ps forgot to update. 619cdbe